src/libraries/Microsoft.PowerFx.Repl/Services/StandardFormatter.cs

```diff
@@ -28,6 +28,10 @@ public class StandardFormatter : ValueFormatter
         // Set to Int32.MaxInt to remove restriction.
         public int MaxTableRows { get; set; } = 10;
 
+        // Append a newline to the end of a formatted table.
+        // If the REPL does not include a newline for the prompt, set this to true for proper spacing.
+        public bool FormatTableNewLine { get; set; } = false;
```

Comment:
Does anybody ever set this to true?
It would be nice if we didn't need a separate flag here - we're basically double capturing this (in Repl.Prompt, and here).

Reply:
No, nobody sets the FormulatTableNewLine flag today.  If Prompt didn't include "\n", then it should be set to true.  In theory the formatter could go look at the REPL Prompt property but that seemed like more dependency than this was worth and currently the formatters are a leaf node and unaware of the hosting REPL which is a good thing.